crates/nargo_cli/tests/test_data_ssa_refactor/constant_return/src/main.nr

```diff
@@ -0,0 +1,13 @@
+// This is a test for simple return with a constant.
+//
+// ACIR generation checks if there is no return 
+// type by checking to see if the return type is a numeric
+// constant with type None.
```

Comment:
I'm not following this comment - it sounds irrelevant to the test in question. Furthermore I'm yet to find the described check happening anywhere.

Reply:
This is what it was doing before the merge, and I'm guessing the dfg is doing this internally when we call `is_return_type`

Reply:
I'll remove it